react/index.d.ts

```diff
@@ -1,5 +1,9 @@
 import { ReadableStore } from '../create-store/index.js'
 
+export interface UseStoreOptions {
+  observeOnly?: string[]
```

Comment:
Too long and hard to remember.

What do you think about `only`?

Reply:
> Too long and hard to remember.
> 
> What do you think about `only`?

only is too uncertain, maybe keys?

Reply:
:+1: